specifyweb/frontend/js_src/lib/components/Attachments/Dialog.tsx

```diff
@@ -16,6 +16,7 @@ import type { SpecifyModel } from '../DataModel/specifyModel';
 import type { Attachment } from '../DataModel/types';
 import { SaveButton } from '../Forms/Save';
 import { Dialog } from '../Molecules/Dialog';
+import { getIIIFUrlFromVersion, IIIFViewer, useIIIFSpec } from './IIIFViewer';
```

Comment:
can you please rename all usages to be `Iiif` instead of `IIIF` or `IiiF`

see https://medium.com/@thomasreggi/enforcing-strict-camelcase-5332c53a7484

tldr: if we always use strict camel case, then there is no ambiguity of whether an acronym should be capitalized or not, and there is consistency (you already used two different variations in this PR

Reply:
fixed